chain-crypto/src/key.rs

```diff
@@ -167,6 +167,9 @@ impl<A: AsymmetricKey> SecretKey<A> {
     pub fn from_binary(data: &[u8]) -> Result<Self, SecretKeyError> {
         Ok(SecretKey(<A as AsymmetricKey>::secret_from_binary(data)?))
     }
+    pub fn security_leak(self) -> A::Secret {
+        self.0
+    }
```

Comment:
Missing a comment on the function to explain what this function does: leak the secret key, giving back the ownership of the secret key. There may not be an appropriate destructor for it anymore.

Reply:
I'd name it `leak_secret` and document the effect. Otherwise I see no problem with adding the method.

Reply:
I don't see a `Drop` implementation for `SecretKey` anyway, though.